lib/inferno/apps/cli/templates/lib/%library_name%/version.rb.tt

```diff
@@ -0,0 +1,3 @@
+module <%= module_name %>
+  VERSION = '0.0.1'.freeze
```

Comment:
Idea: default to `0.0.0`, and then as we like we consider a version of `0.0.0` effectively 'unversioned' (as it was before) and hide it in the UI, etc.  When I first start working on something I kinda hate seeing that 0.0.1 as I consider it 'versionless' until I assign it a version.

Another nice thing is that when the person is prepping for a first release they need to bump this version, which trains them to update a version just prior to release.  And having it be `0.0.0` gives them a hint as far as what the expected version setup should be (vs. an empty string).

Or we could even do `0.0.0-dev` if we wanted to lean into that practice more, but I think even on (g)(10) we found doing that to be more work than necessary.

Reply:
Looks like [semver 2.0](https://semver.org/spec/v2.0.0.html) recommends using 0.1.0 for an initial development release, so I'd argue we start with that, or go with `0.1.0-pre`, or go with `0.0.0` as an 'pre-versioned' option that gets hidden in the UI and should be updated as part of the normal first release cycle.

Reply:
I like `0.0.0`.